test/unittests/main_UnitTest.cpp

```diff
@@ -67,7 +67,6 @@ unsigned int  cprForwardLimit       = 1000;
 bool          noCache               = false;
 char          fwdHost[64];
 char          notificationMode[64];
-unsigned int  connectionMemory      = 64;
```

Comment:
This here was a big mistake. The library depended on a global variable of the executable.
Now, this value is passed to restInit, the way it *must* be. The new variable holding this value is inside the rest library.